updater/src/Updater/SyncLabels/Repos.purs

```diff
@@ -0,0 +1,189 @@
+module Updater.SyncLabels.Repos where
+
+-- spago ls packages | grep 'com/purescript/' | sed -r 's/([ \t])+/ /g' |
+--  \ cut -d ' ' -f 4 | sed 's/"//g' | sed 's;https://github.com/;;' |
+--  \ sed -r 's/.git$//g' | cut -d '/' -f 2 | tr '\n' ' '
+{-
+spago ls packages | grep 'com/purescript/' | sed -r 's/([ \t])+/ /g' |
```

Comment:
Perhaps we should have a script for this, which generates a PureScript module with the relevant repositories included. No need for fancy codegen a la `tidy-codegen`, but we could wrap this up in a script that we can run like `spago run -m Scripts.SyncRepositories` from time to time to make sure we've got up to date libraries. What do you think?

Reply:
Yeah, but perhaps as a separate PR?